phpBB/language/en/email/short/privmsg_notify.txt

```diff
@@ -2,12 +2,10 @@ Subject: New private message has arrived
 
```

Comment:
You didn't change the subject on this one?

Reply:
Yeah, I don't know to what. Is already pretty short. 

Maybe just:

> Subject: Private message has arrived

Reply:
Now I did, this was forgotten.